packages/thirdweb/src/react/web/ui/TransactionButton/index.tsx

```diff
@@ -28,14 +28,77 @@ import { Button } from "../components/buttons.js";
  * ```tsx
  * <TransactionButton
  *   transaction={() => {}}
- *   onTransactionConfirmed={handleSuccess}
- *   onError={handleError}
  *   unstyled
  *   className="bg-white text-black rounded-md p-4 flex items-center justify-center"
  * >
  *   Confirm Transaction
  * </TransactionButton>
  * ```
+ *
+ * Handle errors
+ * ```tsx
+ * <TransactionButton
+ *   transaction={() => ...}
```

Comment:
Let's avoid having incomplete code examples like this `...` - Portal website will not be able to format and/or highlight it properly